jdmkrt/binary-plug/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.xtreemfs.opendmk</groupId>
     <artifactId>opendmk-jdmkrt</artifactId>
-    <version>1.0-b02-SNAPSHOT</version>
+    <version>1.0-b03-SNAPSHOT</version>
```

Comment:
I wouldn't update this version, as no changes have occured.

Reply:
Done